test/models/diffusion/test_song_unet.py

```diff
@@ -63,6 +63,26 @@ def test_song_unet_forward(device):
         atol=1e-3,
     )
 
+    # Same, but with checkpointing enabled
+    model = UNet(
+        img_resolution=64,
+        in_channels=2,
+        out_channels=2,
+        embedding_type="fourier",
+        channel_mult_noise=2,
+        encoder_type="residual",
+        resample_filter=[1, 3, 3, 1],
+        checkpoint_level=4,
+    ).to(device)
+
+    assert common.validate_forward_accuracy(
```

Comment:
Would be great to also add a backward accuracy test. See this as an example: https://github.com/NVIDIA/modulus/blob/main/test/models/graphcast/test_grad_checkpointing.py#L25

Reply:
Rewrote the gradient checkpointing tests as a seperate suite, now includes forward and backward.